hcp/preprocessing.py

```diff
@@ -62,6 +62,42 @@ def apply_ica_hcp(raw, ica_mat, exclude):
     raw._data /= 1e15
 
 
+def apply_ref_meg_residual_fit(raw, decim_fit=100):
```

Comment:
more type checks, more to test :) I think numpy is handling this already
with warnings. Since we're not decimating the data here (and no sfreq gets
updated we're fine.

On Wed, Aug 10, 2016 at 12:03 PM Mainak Jas <notifications@github.com>
wrote:

> In hcp/preprocessing.py
> <https://github.com/mne-tools/mne-hcp/pull/16#discussion_r74217662>:
>
> > @@ -62,6 +62,42 @@ def apply_ica_hcp(raw, ica_mat, exclude):
> >      raw._data /= 1e15
> >
> >
> > +def apply_ref_meg_residual_fit(raw, decim_fit=100):
>
> doc and/or a type check which raises an explicit error if it's not int
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-hcp/pull/16/files/2950dbcd5a63d9b8a4c2550ab1eff8010de5970a#r74217662>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB0fiv1J54lq8EN964TmogrWF8jerKjDks5qeaHxgaJpZM4Jg7kG>
> .
>